### PR TITLE
[13.0][IMP]demo_student:removed comments

### PR DESCRIPTION
**Pull request by khushbu-3**
this pull request removed all unnecessary comments in demo_student module

### CODE REVIEW

**Comment by Dangar-Serpentcs** [comment]
Please improve the description of this pull request.

Always mention the exact module name of changes the code with a version of the module.

**Comment by Dangar-Serpentcs** [comment]
Reference:- https://github.com/OCA/web/pull/1908

File: demo_student/models/student.py [hunk 1]
```diff
@@ -50,15 +48,6 @@ class StudentStudent(models.Model):
...
-    # def button_name(self):
-    #     data = {
-    #         'student.student': self.id,
-    #     }
-    #
-    # return self.env.ref(student.student.print_report_pdf
-    # ').report_action(self, data=data)
     @api.constrains('birthdate', 'code')
     def _check_birthdate(self):
         # print('--------------constrains---------------------',self)
```
Comment (Dangar-Serpentcs) [inline demo_student/models/student.py:53]: Remove print statement

File: demo_student/models/department.py [hunk 2]
```diff
@@ -6,8 +6,7 @@ class DepartmentDepartment(models.Model):
     _description = 'Department'
     _rec_name = 'dept_name'
 
-    dept_ids = fields.One2many('subject.subject', 'sub_id',
-    				string="Department name")
+    dept_ids = fields.One2many('subject.subject', 'sub_id', string="Department name")
     dept_name = fields.Char("Department Name", required="True")
     dept_code = fields.Char("code")
     Department = fields.Char("Department")
```
Comment (Dangar-Serpentcs) [inline demo_student/models/department.py:12]: It should not be a Capital letter field name.

File: demo_student/models/student.py (hunk 1 above)
Comment (khushbu-3) [reply demo_student/models/student.py:53]: ok. i'll remove it

**Comment by Dangar-Serpentcs** [comment]
Please make pull request from different branch

**Pull request closed without merging**